src/init.cpp

```diff
@@ -1269,18 +1269,7 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     // as they would never get updated.
     if (!ignores_incoming_txs) node.fee_estimator = std::make_unique<CBlockPolicyEstimator>();
 
-    assert(!node.mempool);
     int check_ratio = std::min<int>(std::max<int>(args.GetIntArg("-checkmempool", chainparams.DefaultConsistencyChecks() ? 1 : 0), 0), 1000000);
```

Comment:
In the third commit:

I think this should be moved as well, otherwise it is easy to confuse it with some other check_ratio

Edit: I see it is renamed later, but I still think it could make sense to move as well.

Reply:
Also, would it be better to split the peerman move into a separate commit?

Also, it would be best to do a rebase first to pull in the intermediate changes to init.cpp?

Reply:
> I think this should be moved as well, otherwise it is easy to confuse it with some other check_ratio
Done in 976679b251e417d3aa7329807f3d934944ab5252

> Also, would it be better to split the peerman move into a separate commit?
Don't think so, the diff itself isn't that large, and the peerman move only makes sense in the context of moving the mempool and chainman

> Also, it would be best to do a rebase first to pull in the intermediate changes to init.cpp?
Done as of the 133d98f446217f0fa37aa5b067fb74d3ac4e3b01 push